libtrap/README.ifcspec.md

```diff
@@ -25,13 +25,33 @@ Parameters when used as INPUT interface:
 ```
 <hostname or ip>:<port>
 ```
+If you leave hostname/ip blank, IFC assumes you want to use localhost.
```

Comment:
This should be checked again later. `leave out` or `leave blank`?
Ideally, it should be either `<type>:<hostname>:<port>` or `<type>:<port>`.